_shows/72_73/beckett_festival_lunch.md

```diff
@@ -0,0 +1,37 @@
+---
+title: A Beckett Festival - Lunchtimes
+season: In House
+period: Autumn
+playwright: Samuel Beckett
+season_sort: 130
```

Comment:
My reading of this is that Programme C occurs in the evenings of the days that Lunchtimes occur. Therefore the first performance of Lunchtimes is before that of PC, so can the season sorts be swapped?